Hw6/ConsolePrinter.cpp

```diff
@@ -0,0 +1,113 @@
+#include "ConsolePrinter.h"
+
+#include <vector>
+#include <iostream>
+#include <format>
+#include <limits>
+
+#include "HelperTypes.h"
+
+static const std::vector<std::string> Madness = { "Let me tell you buddy! You're out of range!",
+"Are those masks? Are we in the jungle?",
+"Most people hate long lines",
+"Did I ever tell you what the definition of insanity is?",
+"Insanity is doing the exact... same damn thing... over and over again expecting... thing to change...",
+"That. Is. Crazy." };
+
+void ConsolePrinter::showOutRangeMessage()
+{
+	printOutRangeMessage();
+	incrementOutRangeCounter();
+}
+
+void ConsolePrinter::printOutRangeMessage() const
+{
+	std::cout << Madness[m_countOfRange] << "\n";
+}
+
+void ConsolePrinter::incrementOutRangeCounter()
+{
+	if (Madness.size() - 1 > m_countOfRange)
+		++m_countOfRange;
```

Comment:
you can do something like this:
m_countOfRange = (m_countOfRange + 1) % Madness.size();

Reply:
That's great!